src/cortex_cli/cortex_cli.py

```diff
@@ -108,7 +108,13 @@ def _validate_path(ctx: click.Context, param: click.Path, path: str) -> str:
         return new_path
 
 
-@click.group()
+class CortexCliCommand(click.Group):
```

Comment:
Should this class also wrap other global consts that are used as default values in click commands? (`BASE_URL`, `REALM_NAME`, `CLIENT_ID`)

Reply:
`REALM_NAME` and `CLIENT_ID` do not look like something we might need to test (and expose through this "interface"), but `BASE_URL` will simplify tests a bit — `--url` can be removed from all (probably) the tests.